xml/issue3359.xml

```diff
@@ -32,15 +32,14 @@ their insertion date, and update wording and examples in <sref ref="[time.clock]
 <blockquote>
 <pre>
 template&lt;class Duration&gt;
-static sys_time&lt;common_type_t&lt;Duration, seconds&gt;&gt;
-  constexpr span(It first, size_type count);
+  static sys_time&lt;common_type_t&lt;Duration, seconds&gt;&gt;
```

Comment:
This is the only functional change.

Reply:
What's the change here? I see the addition of two leading spaces.....

Reply:
Ah, ok - it's in the line below.